tools/pipelines/build-azure.yml

```diff
@@ -25,7 +25,7 @@ parameters:
 - name: buildToolsVersionToInstall
   displayName: Fluid build tools version (default = installs version in repo)
   type: string
-  default: latest
+  default: "^0.4.4000"
```

Comment:
Should this version be different from the others?

Reply:
This is the version that azure packages are using in the release/azure/1.1 branch. Although I think we could upgrade the version to match other packages I don't think it's necessary